config/settings.yml

```diff
@@ -1074,19 +1077,32 @@
   :custom_logo: false
   :events:
     :disk_usage_gt_percent: 80
+  :heartbeat_timeout: 2.minutes
+  :heartbeat_frequency: 30.seconds
   :host:
   :listening_port: '443'
   :mks_classid: 338095E4-1806-4BA3-AB51-38A3179200E9
   :mks_version: 2.1.0.0
   :monitor_poll: 5.seconds
   :name: EVM
+  :prefetch_max_per_worker: 10
+  :prefetch_max_per_worker_dequeue: 100
+  :prefetch_min_per_worker_dequeue: 10
```

Comment:
I just noticed these 2 new keys were added that weren't there previously.  And the corresponding usage (e.g. https://github.com/ManageIQ/manageiq/pull/13284/files#diff-c0183a7fcb90fec8ded83d618231abe8R65) was changed as well.  I curious as to why?  We should avoid renaming keys in this PR, because that will introduce a data migration.